homeassistant/components/subaru/device_tracker.py

```diff
@@ -0,0 +1,94 @@
+"""Support for Subaru device tracker."""
+from __future__ import annotations
+
+from typing import Any
+
+import subarulink.const as sc
```

Comment:
Since you are only using 3 const, it's probably best to just import those three by themselves.